.github/workflows/test.yml

```diff
@@ -13,4 +13,4 @@ jobs:
       run: docker-compose pull
 
     - name: Build site
-      run: docker-compose run wiki build
+      run: docker-compose run wiki build -v
```

Comment:
Why are we printing the version?

Reply:
`-v` builds the site in verbose mode, not print the version.

This means when building on CI, you can see more information on what may have gone wrong. It's also useful to see progress on the build.